docs/features/traffic-management/alb.md

```diff
@@ -0,0 +1,67 @@
+kind: Ingress
+metadata:
+  annotations:
+    alb.ingress.kubernetes.io/actions.stable-service: |
+      { 
+        "Type":"forward",
+        "ForwardConfig":{ 
+          "TargetGroups":[ 
+            { 
+                "Weight":10,
+                "ServiceName":"canary-service",
+                "ServicePort":"80"
+            },
+            { 
+                "Weight":90,
+                "ServiceName":"stable-service",
+                "ServicePort":"80"
+            }
+            ]
+        }
+      }
+    kubernetes.io/ingress.class: aws-alb
+  name: ingress
+spec:
+  rules:
+    - http:
+        paths:
+          - backend:
+              serviceName: stable-service
+              servicePort: use-annotation
+            path: /*
+```
+This Ingress uses the `alb.ingress.kubernetes.io/actions.stable-service` annotation to define how to route traffic to the various services for the rule with the `stable-service` serviceName instead of sending traffic to the canary-service service. You can read more about these annotations on the official [documentation](https://kubernetes-sigs.github.io/aws-alb-ingress-controller/guide/ingress/annotation/#actions).
+
+## Integration with Argo Rollouts
+There are a couple of required fields in a Rollout to send split traffic between versions using ALB ingresses. Below is an example of a Rollout with those fields:
+
+```yaml
+apiVersion: argoproj.io/v1alpha1
+kind: Rollout
+spec:
+  ...
+  strategy:
+    canary:
+      canaryService: canary-service # required
+      stableService: stable-service  # required
+      trafficRouting:
+        alb:
+           ingress: ingress  # required
+           annotationPrefix: custom.alb.ingress.kubernetes.io # optional
+```
+
+The ingress field is a reference to an Ingress in the same namespace of the Rollout. The Rollout requires this Ingress to modify the ALB to route traffic to the stable and canary Services. Within the Ingress, looks for the stableService within the Ingress's rules and adds an action annotation for that the action. As the Rollout progresses through the Canary steps, the controller updates the Ingress's action annotation to reflect the desired state of the Rollout enabling traffic splitting between two different versions.
+
+Since the ALB Ingress controller allows users to configure the annotation prefix used by the Ingress controller, Rollouts can specify the optional `annotationPrefix` field. The Ingress uses that prefix instead of the default `alb.ingress.kubernetes.io` if the field set.
+
+The Rollout adds another annotation called `rollouts.argoproj.io/managed-alb-actions` to the Ingress to help the controller manage the Ingresses. This annotation indicates which actions are being managed by Rollout objects (since multiple Rollouts can reference one Ingress). If a Rollout is deleted, the Argo Rollouts controller uses this annotation to see that this action is no longer managed, and it is reset to only the stable service with 100 weight.
```

Comment:
> If a Rollout is deleted, the Argo Rollouts controller uses this annotation to see that this action is no longer managed, and it is reset to only the stable service with 100 weight.

I wanted to call out that this behavior is such that we would be leaving around our annotation when the original ingress object did not have it in the beginning. 

I **_think_** this is the correct behavior, since it is safer than deleting the annotation, which could cause downtime, but just wanted to point out that we could be leaving around leftover cruft.